test/Dispatcher/Dispatcher.t.sol

```diff
@@ -350,11 +351,22 @@ contract DispatcherRecvPacketTestSuite is ChannelOpenTestBaseSetup {
             vm.expectEmit(true, true, true, true, address(dispatcherProxy));
             emit RecvPacket(address(mars), channelId, packetSeq);
             vm.expectEmit(true, true, false, true, address(dispatcherProxy));
-            emit WriteAckPacket(address(mars), channelId, packetSeq, AckPacket(true, appAck));
+            emit WriteAckPacket(address(mars), channelId, packetSeq, AckPacket(AckStatus.SUCCESS, appAck));
             dispatcherProxy.recvPacket(IbcPacket(src, dest, packetSeq, payload, ZERO_HEIGHT, maxTimeout), validProof);
         }
     }
 
+    // check that for pluto, recvPacket does not trigger an ack
+    function test_pluto_noAck() public {
+        uint64 packetSeq = 1;
+        Pluto pluto = new Pluto(dispatcherProxy);
+        string memory plutoPort = string(abi.encodePacked(portPrefix, getHexBytes(address(pluto))));
+        IbcEndpoint memory plutoDest = IbcEndpoint(plutoPort, channelId);
+        vm.expectEmit(true, true, true, true, address(dispatcherProxy));
+        emit RecvPacket(address(pluto), channelId, packetSeq);
+        dispatcherProxy.recvPacket(IbcPacket(src, plutoDest, packetSeq, payload, ZERO_HEIGHT, maxTimeout), validProof);
```

Comment:
in addition to the asserting that the recvPacket event was emitted, we also need to test that:
- the `WriteAckPacket` was NOT emitted 
- `_ackPacketCommitment` was NOT set to true

for the first case, we have to be a bit clever, use [expectEmit](https://book.getfoundry.sh/forge/cheatcodes?highlight=emit#cheatcodes) and emit a unique event after the call, such that the expect emit will pass iff `writeAckPacket` event was not emitted (I'd suggest reading expect emit docs first to understand how event testing works, it's a tad unintuitive) 

for second case, you can use something like `findAckPacketCommitmentSlot` in `test/upgradeableProxy/UpgradeUtils.t.sol` to calculate and read from a storage slot in dispatcher proxy, and ensure that that is indeed not set to `true`